README.md

```diff
@@ -1,5 +1,6 @@
 # review-rot
 reviewrot is a CLI tool, that helps to list down open review requests from github, gitlab, pagure and gerrit.
+It can also get open review requests in Phabricator from specified users. 
```

Comment:
It might read more naturally to edit the first line to say `from github, gitlab, pagure, gerrit, and phabricator`.